masterlist.yaml

```diff
@@ -3728,15 +3728,7 @@ plugins:
         util: 'SSEEdit v4.0.3'
   - name:  'True Storms - Obsidian Weathers - Patch .esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/12125/' ]
-    inc:
-      - 'Obsidian_TS_Patch_Luminous.esp'
-      - 'Obsidian_TS_Patch_Spectral.esp'
-      - 'Obsidian_TS_Patch_Luminous_WaterFix.esp'
-      - 'Obsidian_TS_Patch_Spectral_WaterFix.esp '
```

Comment:
Apparently the whitespace at the end went unnoticed for nearly 3 years :P

Reply:
Must have missed it last time I checked for those, a search for `.esp '` `.esm '`  `.esl '` and  `.esp "` `.esm "`  `.esl "` should find most of them.

Reply:
Couldn't find any other instances in any of the masterlists.